app/lib/fullUrlUtils.js

```diff
@@ -0,0 +1,15 @@
+function url(req) {
+  const protocol = req.protocol;
+  const host = req.get('host');
+  const originalUrl = req.originalUrl;
+  // eslint-disable-next-line no-script-url
+  return `${protocol}${host}${originalUrl}`;
```

Comment:
This needs to be include `://` i.e. `${protocol}://${host}${originalUrl}`.
It also looks like the protocol is coming through as `http` which will be because the SSL termination happens at the proxy. It might be worth hardcoding this to be `https://${host}${originalUrl}`. What do you reckon?

Reply:
Yup, seems sensible to me. Done